merge_checks/commit_checks.py

```diff
@@ -71,29 +69,24 @@ def main(head_hash: str, base_ref: str) -> int:
 
     if fixups or squashes:
         print(f"Found {fixups} fixup and {squashes} squash commits!")
-        return 1
+        return False, f"{fixups} fixup and {squashes} squash commits found"
     else:
         print("No fixups or squashes found, check passed!")
 
     if has_merge_commits(head_hash, base_hash):
         print("Branch contains merge commits!")
-        return 1
+        return False, "Contains merge commits"
     else:
         print("Branch does not contain merge commits, check passed!")
 
-    incorrect_commit_messages = has_wrong_commit_message(subjects)
-    if incorrect_commit_messages:
+    if incorrect_commit_messages := has_wrong_commit_message(subjects):
         print(
             f"Found invalid commit message(s), allowed types: {ALLOWED_COMMIT_MESSAGE_TYPES}\n"
             f"{chr(10).join(incorrect_commit_messages)}",
         )
-        return 1
+        return False, "Invalid scope in commit message(s)"
```

Comment:
```suggestion
        return False, "Invalid commit message format found"
```

The regex doesn't only check scopes, right? E.g. missing space should also blow up...

Reply:
Agreed, fixed. a2e3565129222637c8a0841ccf4cdc6927051f17